cfgov/v1/models/snippets.py

```diff
@@ -69,16 +69,28 @@ def __str__(self):
 @python_2_unicode_compatible
 @register_snippet
 class RelatedResource(index.Indexed, models.Model):
-    title = models.CharField(
-        max_length=255
-    )
-    text = RichTextField()
+    title = models.CharField(max_length=255)
```

Comment:
What do you think of renaming these fields to `title_en` etc, so that `trans_title()` below can just be `title()` and `text()` etc ?  I think Django handles it as a renaming and not a new/ deleted field, so it shouldn't cause any migration issues.

Reply:
I was trying to avoid breaking any existing template usages, which would not pass a language value. But on second thought I think a bare template call for .title or .text without passing a language should return the English version if the default value is 'en'. 
If this tests out, I'll swap.

Reply:
@higs4281 I can't find any other references to related_resource on a template other than the answer-page template, but if there are or there's a concern for future templates that don't have access to the language, and `.title` doesn't work, `.title()` certainly should.

Reply:
A bare template call does not work, so using title() and text() would break any current template (or other ) use. It does appears that our answerpage is the only template use, but in troublehsooting today with @nataliafitzgerald we discovered snippet usage reporting can be unreliable. 

There is at least one other use besides in templates: the export_ask_data.py script, which exports related-resource values and would break.

I don't think it's worth chasing this stuff or risking breaking other parts of the site for the sake of a translated field name that will, as you note, probably only be used in the answerpage template. I'd vote for backward compatibility and a `trans_` value name.

Reply:
@higs4281 That makes sense to me.  If I have time after this project, I'd like to go through all the places we have translated fields and try to standardize them, so maybe we can eventually agree on an approach, but this seems great for now.

Reply:
I totally agree one approach is ideal. 

I was down with the simplicity of using the main field names as the translated values, but I'm liking the explicit nature of `trans_` and how it mirrors Django template language.